crates/router/src/connector/stripe/transformers.rs

```diff
@@ -286,6 +286,64 @@ pub struct StripeBankRedirectData {
     pub bank_specific_data: Option<BankSpecificData>,
 }
 
+#[derive(Debug, Eq, PartialEq, Serialize)]
+pub struct AchBankTransferData {
+    #[serde(rename = "owner[email]")]
+    pub email: Email,
+}
+
+#[derive(Debug, Eq, PartialEq, Serialize)]
+pub struct BacsBankTransferData {
+    #[serde(rename = "payment_method_data[type]")]
+    pub payment_method_data_type: StripePaymentMethodType,
+    #[serde(flatten)]
+    pub billing_address: StripeBillingAddress,
```

Comment:
billing_address is already there in `PaymentIntentRequest`, you can use the same